server/pathwayResolver.js

```diff
@@ -187,6 +187,15 @@ class PathwayResolver {
         }
         else {
             // Syncronously process the request
+            return await this.runPathway(args);
+        }
+    }
+
+    async runPathway(args) {
+        if (this.pathway.executeOverride) {
+            return await this.pathway.executeOverride({ args, runAllPrompts: this.promptAndParse.bind(this) });
```

Comment:
... and call this `executePathway` as well.  And check to make sure it's a function here instead of just truthy.